src/main/resources/avro/wip/variationReference.avdl

```diff
@@ -0,0 +1,263 @@
+@namespace("org.ga4gh.wip")
```

Comment:
We abandoned using the graph branch, and actually did the graph stuff as a direct pull request against master, since I found a way to make it backward-compatible.

We probably should make an issue to close that topic branch.